app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -302,13 +303,21 @@ class BrowserActivity : DuckDuckGoActivity() {
         return false
     }
 
+    private fun addBookmark() {
```

Comment:
I think this operation needs some visual feedback to confirm it worked. I pressed it twice thinking it hadn't worked the first time.

from view model back to activity with a "show bookmark added" command, perhaps, which manifests itself as a `Toast` or `Snackbar`?

Reply:
Good call